### PR TITLE
more fine grained language to justify solution

### PR DESCRIPTION
**Pull request by psteinb**
and exclude computational prohibitive answers (suggested by a learner)

Please delete the text below before submitting your contribution. 

---

Thanks for contributing! If this contribution is for instructor training, please send an email to checkout@carpentries.org with a link to this contribution so we can record your progress. You’ve completed your contribution step for instructor checkout just by submitting this contribution.  

Please keep in mind that lesson maintainers are volunteers and it may be some time before they can respond to your contribution. Although not all contributions can be incorporated into the lesson materials, we appreciate your time and effort to improve the curriculum.  If you have any questions about the lesson maintenance process or would like to volunteer your time as a contribution reviewer, please contact Kate Hertweck (k8hertweck@gmail.com).  

---

### CODE REVIEW

**Comment by dafnevk** [comment]
What were the considerations for this change exactly?
I'm not sure I fully agree with it, 'should' is a very normative word to use, but I think this question is open for discussion (although we do provide a 'solution' that we think is the best approach).

**Comment by psteinb** [comment]
Our learners had the impression that the second question referred to the answers of the first. Which they felt was misleading.

**Comment by dafnevk** [comment]
Ok I see. Different suggestion: use `would` instead of `could`?

File: _episodes/01-introduction.md [hunk 1]
```diff
@@ -88,7 +88,7 @@ The following could technically be achieved using Deep Learning, but it would pr
 
 > ## Deep Learning Problems Exercise
 >
-> Which of the following could you apply Deep Learning to?
+> Which of the following should you apply Deep Learning to?
```
Comment (psteinb) [inline _episodes/01-introduction.md:91]:
```suggestion
> Which of the following would you apply Deep Learning to?
```

Comment (psteinb) [reply _episodes/01-introduction.md:91]: @dafnevk I am commenting on my own PR. github offers too many options to change my own code. :-)

**Pull request merged**